package.json

```diff
@@ -7,6 +7,20 @@
   "scripts": {
     "build": "yarn workspaces foreach run build",
     "lint": "yarn workspaces foreach run lint",
-    "test": "yarn workspaces foreach run test"
+    "test": "jest"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.17.2",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-typescript": "^7.16.7",
```

Comment:
We have TS, Babel shouldn't be needed.

Reply:
Ah yeah, I was having a hard time setting up JS files with TS.

Reply:
No Babel please, we already transform via TypeScript :smiley: